### DISCUSSION TITLE
Why did the emojis I see change to bland gray font characters?

### DISCUSSION THREAD

**Discussion by jay** [community/community#47821] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm using the latest Chrome for Desktop and the colored png emojis that I normally see in posts have been replaced by bland character based font emojis. I have disabled all my extensions and it hasn't made a difference. Also, I looked at a [user collection of all github emojis](https://gist.github.com/rxaviers/7360908) and I see pretty much the same thing, most are gray font characters and not the regular graphics. If I use a clean browser profile where I am not logged in I will see the graphics.

![Capture](https://user-images.githubusercontent.com/965580/219788166-e9d335cc-687d-4375-b10d-1d590d096507.PNG)

![comparison](https://user-images.githubusercontent.com/965580/219790576-2b44488b-06a8-4079-8c47-d3c0ab9d3311.png)

**Comment by radar-blip** [comment]
Don't know but I like it. 😄
